cli/src/config.rs

```diff
@@ -62,7 +62,7 @@ impl Default for Config {
         registries.insert(
             registry_name.to_string(),
             Registry {
-                base_url: url::Url::parse("https://database.dev")
+                base_url: url::Url::parse("https://xmuptpplfviifrbwmmtv.supabase.co")
```

Comment:
Purely an aesthetic thing, so it doesn't really matter, but can we use `https://api.database.dev` since it's configured as a custom domain on the project?

Reply:
Great idea. Done.